src/qililab/platform/platform.py

```diff
@@ -1367,3 +1431,17 @@ def compile(
         )
 
         return compiled_programs, final_layout
+
+    def draw_oscilloscope_platform(self, qprogram: QProgram, averages_displayed: bool = False):
```

Comment:
We could rename this to `draw_oscilloscope`.

Reply:
Or we could name this `draw` globally. (I mean we don't have any other draw method to differentiate from)